src/Datepicker/DatesList.tsx

```diff
@@ -78,11 +84,11 @@ interface IListItem {
   disabled?: boolean
 }
 
-const ListItem = styled.li<IListItem>`
+const ListItem = styled.button<IListItem>`
```

Comment:
this should probably be `ListButton` and hte `container` can simply be a `div` rather than a `ul` as I don't think `button` as direct children of a `ul` semantically correct 😄

Reply:
👀 👀 👀 , just pushed this change